### PR TITLE
Update paket

### PR DESCRIPTION
**Pull request by JarnoNijboer**
Updated paket for fsprojects/Paket#3093

### CODE REVIEW

File: help/markdown/fake-dotnetcore.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Just install the corresponding package for your system:
 choco install fake -pre
 ```
 
-now you can use
```
Comment (matthid) [inline help/markdown/fake-dotnetcore.md]: Just a note for your editor settings, whitespace at line endings is significant for markdown files (at least AFAIK)

File: paket.lock [hunk 2]
```diff
@@ -1699,54 +1699,54 @@ NUGET
...
+      System.Runtime.Serialization.Formatters (>= 4.3) - restriction: || (&& (== net46) (== netcoreapp2.0)) (&& (== net46) (== netstandard2.0)) (&& (== net46) (< net20)) (&& (== netcoreapp2.0) (< netstandard2.0)) (== netstandard1.6)
+      System.Runtime.Serialization.Primitives (>= 4.3) - restriction: || (&& (== net46) (== netcoreapp2.0)) (&& (== net46) (== netstandard2.0)) (&& (== net46) (< net20)) (&& (== netcoreapp2.0) (< netstandard1.3)) (&& (== netcoreapp2.0) (< netstandard2.0)) (== netstandard1.6) (&& (== netstandard2.0) (< netstandard1.3))
+      System.Xml.XmlDocument (>= 4.3) - restriction: || (&& (== net46) (== netcoreapp2.0)) (&& (== net46) (== netstandard2.0)) (&& (== net46) (< net20)) (&& (== netcoreapp2.0) (< netstandard2.0)) (== netstandard1.6)
     Octokit (0.29)
       NETStandard.Library (>= 1.6) - restriction: || (&& (== net46) (< net45)) (== netcoreapp2.0) (== netstandard1.6) (== netstandard2.0)
     Paket.Core (5.145.1)
```
Comment (matthid) [inline paket.lock]: Here it seems to be missing?

**Comment by matthid** [comment]
Thanks a lot! Yes we need a new release with this :)

File: help/markdown/fake-dotnetcore.md (hunk 1 above)
Comment (JarnoNijboer) [reply help/markdown/fake-dotnetcore.md]: I checked my settings in Code and trim white space is turned on. Didn't know it is significant for MD.

File: paket.lock (hunk 2 above)
Comment (JarnoNijboer) [reply paket.lock]: I think that was a old commit. I have done a push -f ;)

**Pull request merged**